src/js/map/boundary_type_box.js

```diff
@@ -0,0 +1,161 @@
+import {Observable} from "../utils";
+
+let selectElement = $('.map-geo-select')[0];
+let selectedOption = null;
+let optionWrapper = null;
+let optionItem = null;
+let cookieName = 'boundary_selection';
+
+export class BoundaryTypeBox extends Observable {
+    constructor(preferred_children) {
+        super();
+
+        this.preferred_children = preferred_children;
+
+        this.prepareDomElements();
+    }
+
+    prepareDomElements = () => {
```

Comment:
isn't it better to seperate it as a function and just calling that function in constructor instead of putting 3 more lines into constructor? I always find this much clearer. also we don't know what else we would need to put into constructor in the future so it may get complex and I think better to make it as short as we can from the start.

also in all the other classes, the function that works with dom elements on the page load is called `prepareDomElements`. should I still change it?